docs/get-started/example-solutions/_index.rst

```diff
@@ -5,8 +5,13 @@
 ##########
 
 Get started quickly by using an example machine learning model that has been converted to
-Determined's APIs. Visit the ``examples/`` subdirectory of the `Determined GitHub repo
-<https://github.com/determined-ai/determined/tree/master/examples>`__ or download the link below.
+Determined's APIs. To view code examples maintained by the team, visit the ``examples/``
+subdirectory of the `Determined GitHub repo
+<https://github.com/determined-ai/determined/tree/master/examples>`__ or use one of the download
+links below.
+
+For a large collection of community-contributed examples, visit the `determined-examples repo
+<https://github.com/determined-ai/determined-examples/>`__.
```

Comment:
##########

Start with an example machine learning model converted to
Determined's APIs. Code examples are in the ``examples/``
subdirectory of the `Determined GitHub repo
<https://github.com/determined-ai/determined/tree/master/examples>`__. Download links are below.

For more examples, visit the `determined-examples repo
<https://github.com/determined-ai/determined-examples/>`__.